bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/target/env/Asterism.scala

```diff
@@ -0,0 +1,63 @@
+package edu.gemini.spModel.target.env
+
+import edu.gemini.spModel.core.Coordinates
+import edu.gemini.spModel.target.SPTarget
+import edu.gemini.shared.util.immutable.{Option => GOption}
+import edu.gemini.shared.util.immutable.ScalaConverters._
+import edu.gemini.skycalc.{Coordinates => SCoordinates}
+
+
+import java.time.Instant
+
+import scalaz.NonEmptyList
+
+/** Collection of stars that make up the science target(s) for an observation,
+  * along with any configuration details unique to the instrument in use.
+  */
+trait Asterism {
+
+  /** All targets that comprise the asterism.  There must be at least one target.
+    */
+  def targets: NonEmptyList[SPTarget]
+
+  /** Slew coordinates and AGS calculation base position.  By default this
+    * will be a computation that finds the coordinate minimizing the maximum
+    * distance to any target in the asterism.
+    */
+  def basePosition(time: Instant): Option[Coordinates] =
+    ???  // we should supply the "smallest-circle" implementation here
```

Comment:
This is used all over the place so it will be obvious if it remains unimplemented. Implementing it is one of the next tasks.